.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,18 @@
+name: Nightly Tests
+
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: "0 5 * * 2,6"  # cron is UTC, this translates to 10 PM PST Mon and Fri.
```

Comment:
We used to run different API levels on different days of the week but it's a lot more efficient to just run all of them (in parallel on Firebase Test Lab) a couple times a week.